spec/models/comment_spec.rb

```diff
@@ -0,0 +1,7 @@
+require_relative '../rails_helper'
+RSpec.describe Comment, type: :model do
+  subject { Comment.new(Text: 'follow the spirit') }
+  it 'expect to respond to the method update_comment_counter' do
+    expect(subject).to respond_to(:update_comment_counter)
+  end
```

Comment:
- [ ] How sure are we that the counter has been increased? We want to be sure that the comments counter has been increased by 1.

something like: `expect(subject.update_comment_counter).to_eq 1`